src/main/java/frc/robot/commands/WaitForShooterRPMCommand.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.IntSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter;
+
+public class WaitForShooterRPMCommand extends CommandBase {
+  /** Creates a new WaitForShooterRPMCommand. */
+  public WaitForShooterRPMCommand(Shooter shooter, IntSupplier rpmSupplier, double tolerance) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(shooter);
```

Comment:
This is the error we were getting last night when we deployed the the latest code form this branch. This WaitForShooterRPMCommand doesn't need to require the shooter subsystem as it fights with the RunShooterCommand for ownership. You can think of this command more of listening, instead of controlling the subsystem. So therefore we don't need to require the shooter subsystem in this command.

```
Error at frc.robot.commands.ShootCommandGroup.<init>(ShootCommandGroup.java:28): Unhandled exception: java.lang.IllegalArgumentException: Multiple commands in a parallel group cannotrequire the same subsystems
        at edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup.addCommands(ParallelDeadlineGroup.java:69)
        at edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup.<init>(ParallelDeadlineGroup.java:39)
        at frc.robot.commands.ShootCommandGroup.<init>(ShootCommandGroup.java:28)
        at frc.robot.RobotContainer.configureButtonBindings(RobotContainer.java:125)
        at frc.robot.RobotContainer.<init>(RobotContainer.java:55)
        at frc.robot.Robot.robotInit(Robot.java:30)
        at edu.wpi.first.wpilibj.TimedRobot.startCompetition(TimedRobot.java:107)
        at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:373)
        at edu.wpi.first.wpilibj.RobotBase.lambda$startRobot$0(RobotBase.java:443)
        at java.base/java.lang.Thread.run(Thread.java:829)

```

I recreated the error by running our code in simulation.

Reply:
463f5e4d04767b600c479901454f667177027257